presto-product-tests/src/main/java/io/prestosql/tests/hive/TestAvroSchemaUrl.java

```diff
@@ -46,6 +47,8 @@
     public void setup()
             throws Exception
     {
+        // TODO move Avro schema files to classpath
```

Comment:
What is the issue here? I am afraid this is not the only one such test?

Reply:
Yes, I am afraid so too.
I started setting up SSH client, which would address this in a generic manner, but didn't finish.